### PR TITLE
Add openssl version to version command

### PR DESCRIPTION
**Pull request by dangra**
Motivated by #1429
 
```
$ scrapy version -v
Scrapy  : 1.1.0dev1
lxml    : 3.4.4.0
libxml2 : 2.9.2
Twisted : 15.3.0
Python  : 3.4.3 (default, Mar 25 2015, 17:13:50) - [GCC 4.9.2 20150304 (prerelease)]
OpenSSL : 0.15.1
Platform: Linux-4.1.4-1-ARCH-x86_64-with-arch
```

### CODE REVIEW

**Comment by dangra** [comment]
`OpenSSL.version.__version__` is available (at least) since 0.12 (ubuntu 12.04)

File: scrapy/commands/version.py [hunk 1]
```diff
@@ -31,6 +32,7 @@ def run(self, args, opts):
             print("libxml2 : %s" % libxml2_version)
             print("Twisted : %s" % twisted.version.short())
             print("Python  : %s" % sys.version.replace("\n", "- "))
+            print("OpenSSL : %s" % OpenSSL.version.__version__)
```
Comment (kmike) [inline scrapy/commands/version.py]: Is it a version of OpenSSL Python library or a version of OpenSSL system library?

Comment (dangra) [inline scrapy/commands/version.py]: I updated the code minutes ago to make it clear it is pyOpenSSL

Comment (dangra) [inline scrapy/commands/version.py]: Ah.. not pushed to my fork, now it is. 

Still I found it is possible to get OpenSSL version too but only on recent pyOpenSSL versions

**Comment by dangra** [comment]
Latest output:

```
Scrapy    : 1.1.0dev1
lxml      : 3.4.4.0
libxml2   : 2.9.2
Twisted   : 15.3.0
Python    : 3.4.3 (default, Mar 25 2015, 17:13:50) - [GCC 4.9.2 20150304 (prerelease)]
pyOpenSSL : 0.15.1
OpenSSL   : OpenSSL 1.0.2d 9 Jul 2015
Platform  : Linux-4.1.4-1-ARCH-x86_64-with-arch
```

**Comment by dangra** [comment]
Alternative output 

```
Scrapy    : 1.1.0dev1
lxml      : 3.4.4.0
libxml2   : 2.9.2
Twisted   : 15.3.0
Python    : 3.4.3 (default, Mar 25 2015, 17:13:50) - [GCC 4.9.2 20150304 (prerelease)]
pyOpenSSL : 0.15.1 (OpenSSL 1.0.2d 9 Jul 2015)
```

which one do you prefer? 

I am trying to avoid code refactoring for now

**Comment by kmike** [comment]
I like both outputs.

**Comment by dangra** [comment]
I settled on the last one, I'll squash and merge.

**Pull request merged**